Cargo.toml

```diff
@@ -336,7 +336,8 @@ zeta = { path = "crates/zeta" }
 #
 
 aho-corasick = "1.1"
-alacritty_terminal = "0.24"
+# TODO(#18342): Update to version 0.25 from crates.io when it is released.
```

Comment:
I tagged the TODO with a GitHub issue to keep the context and remember to clean up this dependency when new Alacritty crate is released.
Is there any convention in the codebase for such tagging?

Reply:
I think `TODO(..)` is a convention, at least there's a bunch if Windows-related items like that, or just `TODO ...` is also used.